### PR TITLE
2498 - Add RSS feed for preprints

### LINKED ISSUE #2498: RSS support should be extended to preprints/repository
**Is your feature request related to a problem? Please describe.**
As a preprints server administrator, I would like to have an RSS feed to conveniently share the latest published preprints. 

**Describe the solution you'd like**
Extend RSS support to preprints/repository.

### PR DESCRIPTION
**Pull request by hardyoyo**
fixes #2498

### CODE REVIEW

File: src/rss/urls.py [hunk 1]
```diff
@@ -6,7 +6,8 @@
 from rss import views
 
 urlpatterns = [
-    url(r'^', views.LatestArticlesFeed(), name='rss_articles'),
```
Comment (ajrbyers) [inline src/rss/urls.py:9]: A slight hiccough with this change: anyone who is subscribed to this URL will stop seeing new content.

**Review by ajrbyers — CHANGES_REQUESTED** [review]
One change suggested.

File: src/rss/urls.py (hunk 1 above)
Comment (ajrbyers) [reply src/rss/urls.py:9]: Ah, just reorder them so the LatestArticlesFeed one comes last in the list.

Comment (hardyoyo) [reply src/rss/urls.py:9]: Ah, it *does* work if I move it to the end of the URLs, I'll change that

**Review by mauromsl — APPROVED** [review]

**Pull request merged**